ilastik/applets/dataSelection/addFileButton.py

```diff
@@ -77,3 +78,12 @@ def __init__(self, parent, new=False):
                     self.addRemoteVolumeRequested.emit)
 
         self.setMenu( menu )
+
+    @property
+    def index(self):
+        return self.__index
```

Comment:
the double undescore `__` is bad practice imo. Single underscore is enough to indicate one should not modify the variable from the outside...